### DISCUSSION TITLE
Plugin crashed after dumb trying..

### DISCUSSION THREAD

**Discussion by KeeganJin** [WasmEdge/WasmEdge#2820] [Q&A]
Hi all, 

I tired the wasi-nn example [https://github.com/second-state/wasmedge-rustsdk-examples/tree/main/object-detection-via-wasinn](url) after following the instructions, **it worked,** but when I tried to set the args in the crate for setting up Wasi module by doing: 

`    let dir_mapping = &String::from(".:.");` 
`    let wasm_file = &String::from("wasmedge-wasinn-example-mobilenet-image.wasm");`
`    let model_bin = &String::from("mobilenet.pt");` 
`    let image_file = &String::from("input.jpg");`  
without using the command line, I got the error.
`load plugin
[2023-09-18 14:02:49.202] [error] loading failed: invalid path, Code: 0x20
[2023-09-18 14:02:49.203] [error]     load library failed:libtorch.so: cannot open shared object file: No such file or directory
Error: Plugin(NotFound("wasi_nn"))`.
**I think the dir_mapping could be wrong? if so, what would be right?**

But when I wanted to switch back to the original code of the example, which gets args from the command line and runs 
`cargo run -- .:. wasmedge-wasinn-example-mobilenet-image.wasm mobilenet.pt input.jpg`
it shows the same error:
`load plugin
[2023-09-18 14:15:52.794] [error] loading failed: invalid path, Code: 0x20
[2023-09-18 14:15:52.795] [error]     load library failed:libtorch.so: cannot open shared object file: No such file or directory
Error: Plugin(NotFound("wasi_nn"))`
**I think my trying somehow broke the Plugin path because it worked fine in the very beginning.**
I met this issue several times, the re-installation always helps, but that's quite time-consuming and I don't know how it happened. 
Can anyone help with that, thanks in advance.
BR

**Comment by KeeganJin** [comment]
I think it's an issue with VSCode, it works fine in the Terminal. :)

**Comment by hydai** [comment]
Closing this, since the issue is not a real issue.
